### PR TITLE
Add 'Motivation' section to README

### PR DESCRIPTION
**Pull request by saneyuki**
Depend on https://github.com/nekoya/popuko/pull/28

### CODE REVIEW

**Comment by saneyuki** [comment]
@nekoya r?

**Comment by saneyuki** [comment]
@nekoya r?

File: README.md [hunk 1]
```diff
@@ -11,6 +11,18 @@
 - Almost reimplementation of [homu](https://github.com/barosl/homu).
 
 
+## Motivation
+
+[homu](https://github.com/barosl/homu) is the super great operation bot for development on GitHub
+and it supports a lot of valuable features: merge pull request into the latest upstream, try to testing on TravisCI,
+and more. But its development is not in active now. And also Mozilla's servo team maintains
+[their forked version of homu](https://github.com/servo/homu). But it is developped for their specific usecase.
```
Comment (nekoya) [inline README.md]: s/developped/developed/

File: README.md [hunk 2]
```diff
@@ -11,6 +11,18 @@
...
+Not for other projects.
+
+This project intent to re-implement homu with minimum features to support our projects for work including non-public activities,
+and to simplify to deploy this bot. This is why we development this project
```
Comment (nekoya) [inline README.md]: I'm sorry for trifling comment.

`we development` is based on S+N, I prefer more colorful expression, for example `we have developed`.

**Comment by nekoya** [comment]
I agree to the soul of this Motivation at all.

**Comment by saneyuki** [comment]
@nekoya fixed.

**Comment by saneyuki** [comment]
@popuko r=nekoya

**Comment by popuko** [comment]
Try to merge this pull request which has been approved by `saneyuki`

**Pull request merged**